proposals/Level-1.md

```diff
@@ -78,94 +77,69 @@ instruction. Thrown exceptions are handled as follows:
-resumable events.
-
-The `type signature` of an event type is an index into the `type` section.  The
-type signature is defined by a function type. The parameter list of the type
-signature defines the list of values returned to the corresponding enclosing
-catch. The result type defines the list of values to be used when resuming on an
-event. For exceptions, that can't resume, the result type must always be type
-'void'.
-
-Note: Resumable events may not use the `throw` instruction. Rather, they may use
-a new (yet to be defined) `yeild` instruction that passes value(s) to its
-enclosing contexts. Such an instruction would allow the calling context to
-return values back, and resume with the instruction following the yeild
-instruction, based on the values returned.
+event type corresponds to an exception. Future extensions my add additional
+event specifiers.
 
-Within a module, event types are identified by an `event index` to the [event
-index space](#event-index-space). Each event index refers to a corresponding
-runtime tag that uniquely identifies the corresponding event to be handled.  The
-corresponding runtime tag is called an `event tag`.
-
-Event types are numbered numerically, starting with the import section, and then
-followed by event types in the event section.  Each event type within a module
-are associated with unique event tags.
+The type signature of an event type is an index to the corresponding function
+signature defined in the `type` section. The parameter of the function signature
+defines the list of values associated with the event. The return value(s) of the
+signature defines what is passed back to yeilding instruction so that execution
+may resume.
 
-Event types can also be exported by adding appropriate descriptors in the export
-section of the module. Each imported/exported event index must be named to
-reconcile the corresponding (runtime) event tag.
+Within a module, event types are identified by an `event index` to the [event
+index space](#event-index-space). This (module specific) index identifies the
+corresponding identifie the event type to apply.
 
 ### Exceptions
 
-An `exception` is a WebAssembly event that can be thrown and caught.  That is,
-an `exception type` is an event type whose event specifier is 0.  The `type
-signature` of the exception is the corresponding type signature of the event
-type. That is, the parameter list of the type signature defines the list of
-values associated with the exception, and the result type must be type `void`.
-
-Exceptions are identified by the by the corresponding event index, and is called
-an `exception index`. The corresponding (runtime) event tag is called the
-`exception tag`.  Like all event types, each exception type within a module
-(i.e. in the event and import sections) are associated with unique exception
-tags.
+An `exception` is an internal construct in WebAssembly that is maintained by the
```

Comment:
Same comment here about "maintained by embedder". Perhaps avoid duplicating this text.